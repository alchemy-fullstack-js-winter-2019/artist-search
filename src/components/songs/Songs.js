import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
// import Song from './Song';
// import getArtistSongs from '../../services/getArtistsApi';


class Songs extends PureComponent {
    static propTypes = {
      songs: PropTypes.array
    };

    render() {
      return (
        <ul>
          <h2>Hello world</h2>
        </ul>
      );
    }
  
}

export default Songs;
