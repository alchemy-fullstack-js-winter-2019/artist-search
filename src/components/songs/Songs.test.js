import React from 'react';
import renderer from 'react-test-renderer';
import Songs from './Songs';

describe('Songs', () => {
  it('matches a snapshot', () => {
    // const works = [
    //   {
    //     id: '02b2de82-9f31-3683-b112-419e1b86b1c5',
    //     title: 'Beans'
    //   },
    //   {
    //     id: '05ad4550-a38e-3cda-972c-900b3d335724',
    //     title: 'Carmen: Acte I, No. 5. Havanaise : « L’amour est un oiseau rebelle » (Carmen, chœur)'
    //   }
    // ];
    const tree = renderer.create(
      // <Songs works={works}/>
      <Songs />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
