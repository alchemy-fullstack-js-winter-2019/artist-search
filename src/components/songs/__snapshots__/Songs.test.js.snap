// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Songs matches a snapshot 1`] = `
Array [
  <button
    onClick={[Function]}
  >
     Get Songs
  </button>,
  <h1>
     
     
  </h1>,
]
`;
